conventional-version/src/index.js

```diff
@@ -22,7 +22,10 @@ const run = async () => {
     const githubRef = process.env.GITHUB_REF;
     core.info(`GITHUB_REF: ${githubRef}`);
 
-    const branchName = branchinfo.getBranchName(githubRef);
+    const githubHeadRef = process.env.GITHUB_HEAD_REF;
+    core.info(`GITHUB_HEAD_REF: ${githubHeadRef}`);
```

Comment:
Is this debug output or supposed to always be logged?

Reply:
It aligns with the logging on line 23, so I thought it made sense to keep the logging style. I think it's good to have around since the contents of GITHUB_REF and GITHUB_HEAD_REF doesn't follow the documentation at all.